man/man5/p11sak_defined_attrs.conf.5.in

```diff
@@ -0,0 +1,40 @@
+.TH P11SAK_DEFINED_ATTRS.CONF 5 "September 2021" "@PACKAGE_VERSION@" "openCryptoki"
+.SH NAME
+p11sak_defined_attrs.conf \- Configuration file for p11sak list-key command.
+
+.SH DESCRIPTION
+p11sak uses a configuration file at /etc/opencryptoki/p11sak_defined_attrs.conf
+
+This is a text file that contains information used to configure
+non standard attributes that shall be printed with the p11sak list-key command. 
+
+.SH SYNTAX
+
+.P
+Each attribute description is composed of the attribute title, brackets and three key-value pairs.
+e.g.:
+
+ attribute
+ {
+     name = CKA_IBM_RESTRICTABLE
+     id = 0x80010001
+     type = CK_BBOOL
+ }
+
+All three keywords name, id, type are required to define an attribute.
+
+The name can be anything as long as it is a bare word. 
+The id's are defined in /opencryptoki/usr/include/pkcs11types.h and can be 
```

Comment:
That sentence will lead to people trying to open `/opencryptoki/usr/include/pkcs11types.h` which fails for obvious reasons.  Also this header might only be installed if the development package of opencryptoki is installed (not sure about that).  So referencing it here is not a good idea.

Most of the attributes are also defined in the PKCS11 standard and the rest are IBM-specific extensions.  Maybe use this.  Or reference to the opencryptoki sources?

Reply:
Done